#ifndef MAINWINDOW_H
#define MAINWINDOW_H

#include <QMainWindow>
#include <QtWebEngineWidgets>
#include "webpage.h"


QT_BEGIN_NAMESPACE
namespace Ui { class MainWindow; }
QT_END_NAMESPACE

class MainWindow : public QMainWindow
{
    Q_OBJECT

public:
    MainWindow(QWidget *parent = nullptr);
    ~MainWindow();
public slots:
    void loadFinished(bool x);
protected:
    void  showEvent(QShowEvent *event);
    void resizeEvent(QResizeEvent * event);

private slots:
    void on_splitter_splitterMoved(int pos, int index);

    void on_pushButton_clicked();

private:
    WebPage * webPage;
    QWebEngineView * htmlView;
    Ui::MainWindow *ui;
};
#endif // MAINWINDOW_H
